packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -202,8 +202,8 @@ export class SpinButton extends BaseComponent<ISpinButtonProps, ISpinButtonState
             type='text'
             role='spinbutton'
             aria-labelledby={ label && this._labelId }
-            aria-valuenow={ !Number.isNaN(Number(value)) ? Number(value) : undefined }
-            aria-valuetext={ Number.isNaN(Number(value)) ? value : undefined }
+            aria-valuenow={ !isNaN(Number(value)) ? Number(value) : undefined }
```

Comment:
It would be more robust to check if Number.isNaN is a defined function and if so, to use it rather than rely on the less reliable, global `isNaN` function all the time.

Reply:
@cliffkoh, this is the only usage of Number.isNaN that I could find in fabric, everywhere else uses isNaN